corehq/apps/data_dictionary/management/commands/populate_case_prop_groups.py

```diff
@@ -22,8 +22,9 @@ def handle(self, **options):
 
 def populate_case_prop_groups(domain):
     filter_kwargs = {"case_type__domain": domain}
-    case_props = CaseProperty.objects.filter(**filter_kwargs)
+    case_props = CaseProperty.objects.exclude(group__exact="").filter(**filter_kwargs)
```

Comment:
should this be group_id__exact?

Reply:
the `group` field is blank i.e empty string for all the case properties that don't have a group, `group__exact=""` filters them. I can add one more statement for filtering all the fields that were already assigned a group_id this will make this command idempotent.

Reply:
Is there a difference in how the query is run for `group_id__exact` and `group__exact`?

Reply:
we can use `group_id__isnull=True` in the `filter` method to filter out all the records which have a null group_id.

Reply:
[f3a51e3](https://github.com/dimagi/commcare-hq/pull/32799/commits/f3a51e32da7363f6d7a48356b39b60f7063515f6)